santibout/sun-synergy
# [EXAMPLE PR] COD-36: Add login form

### Description
Adds material UI along with Button + TextInput components and a Login form to the index page. See individual commits for more details.

### Links
https://material-ui.com/

### How to Test
1) Pull down the branch `jlam/cod-36-login-form` (this is implied, you don't need to include this instruction)
2) `npm i`
3) `npm run dev`
4) Visit `localhost:3000
5) Enter an email and password
6) Press submit and verify that the email and password are logged in the JS console
